torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -638,9 +709,13 @@ FusionKernelRuntime* FusionKernelRuntimeCache::getRtByHeuristics(
   if (is_segmented_) {
     heuristics = segmented_fusion_->makeHeuristics(inputs);
   } else {
+    SchedulerRuntimeInfo runtime_info(complete_fusion_, inputs, true);
     auto evaluator = executor_utils::bindFusionInputs(inputs, complete_fusion_);
```

Comment:
We are not using evaluator for anything afterwards. it's not clear to me how we are hooking it up for the heuristics later.

Reply:
Wasn't we creating a evaluator in the previous statement, so this one should be removed?

Reply:
Good catch. Removed the redundant one.